app/(gcforms)/[locale]/(form administration)/form-builder/components/shared/right-panel/logic/GroupSelect.tsx

```diff
@@ -23,15 +23,18 @@ export const GroupSelect = ({
   return (
     <div className="mb-4">
       <label className="mb-2 block text-sm" id={labelId}>
+        {selected === "exit" && <span className="sr-only">{t("disabled")}</span>}{" "}
```

Comment:
Not an ideal solution but it should work. Why this solution?
-disabling a form field does not announce this to AT
-using our live region util would be awkward because the DOM change already announces the label
-adding the "disabled" text at the end of the label announces awkwardly in between the label update and the dropdown update
-this left putting the "disabled" text at the beginning of the label only "visible" to AT